codice/ddf
# DDF-4222 Allow user to enter lat/lon coordinates into the gazetteer search box

#### What does this PR do?
Gives the suggestion service the ability to parse latitude/longitude coordinates from a search string
#### Who is reviewing it? 
@Lambeaux @alanchampion @Bdthomson
#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
#### How should this be tested?
If testing before [DDF-4200](https://github.com/codice/ddf/pull/3847) is merged, use the anyGeo Keyword search. 
![image](https://user-images.githubusercontent.com/39737329/46978786-15d44880-d08d-11e8-86ba-001182bb498f.png)
If testing after, use the gazetteer search box and disable online gazetteer.
![image](https://user-images.githubusercontent.com/39737329/46978872-37353480-d08d-11e8-92d0-cfe1cddbffbf.png)
Test a variety of latitude/longitude coordinates, including
- invalid numbers 
- latitude w/o longitude and vice versa 
- letters and other symbols intermixed
- negative/non-negative
- decimals
and verify that the correct coordinates are suggested
#### Any background context you want to provide?
Depends on [DDF-4200](https://github.com/codice/ddf/pull/3847)
#### What are the relevant tickets?
[DDF-4222](https://codice.atlassian.net/browse/DDF-4222)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
